frontend/src/pages/accounts/profile.module.scss

```diff
@@ -131,6 +132,7 @@
     gap: $spacing-md;
 
     h3 {
+      @include font-mozilla;
```

Comment:
Hey @Vinnl trying to get your take on this, for some reason this line doesn't load the appropriate [Zilla Slab font](https://protocol.mozilla.org/fundamentals/typography.html#typeface-zilla-slab). I tried to import the font face in the code base, following the pattern of Inter UI and Metropolis. However, this is still not working. On the contrary, `@include font-firefox` works just fine.

Reply:
Heh, you had me stumped there for a bit, since you did all the right things. But there are two things going wrong, both in the `zilla-slab.scss` file:

1. The URLs to the fonts are wrong: they refer to `/fonts/ZillaSlab/`, but the folder you created in `frontend/public/fonts` is called `Zilla_Slab` instead of `ZillaSlab`.
2. The name of the `font-family` does not align with the names used by Protocol. If you inspect the styles of this `<h3>`, you'll see that it refers to "Zilla Slab", whereas in `zilla-slab.scss`, it matches `font-family: "ZillaSlab"` (i.e. that should be `font-family: "Zilla Slab"`).

![image](https://user-images.githubusercontent.com/4251/162728009-4a38d7ec-8cba-4800-8873-d59c0b9adeaf.png)

When I fix those, I do get the correct font:

![image](https://user-images.githubusercontent.com/4251/162728194-395c145c-5b6c-438a-9c2d-e48130be6854.png)

![image](https://user-images.githubusercontent.com/4251/162728316-54426d15-ae96-4e2a-83f9-8db80300c826.png)

(Also: we're using _three_ different fonts? Hoowee!)

Reply:
Whoop, thanks for catching that!